tests/unit/async-test-waiter-test.ts

```diff
@@ -1,4 +1,10 @@
-import { getPendingWaiterState, getWaiters, TestWaiter, Token, _reset } from 'ember-test-waiters';
+import {
```

Comment:
We should move these tests to directly use `buildWaiter`

Reply:
Can do.